client/src/modules/list/components/Items/ListItemDescription/ListItemDescription.jsx

```diff
@@ -123,13 +127,15 @@ class ListItemDescription extends PureComponent {
     const {
       currentUser: { id: userId, name: userName },
       disabled,
+      intl: { formatMessage },
       itemId,
       match: {
         params: { id: listId }
       },
       name,
       updateListItem
     } = this.props;
+    const formattedName = formatName(userName, formatMessage);
```

Comment:
do we have to call `formatName` on the current User name? current user can not be anonymous...